From 69010f05559b6d7d7818a118485be545a80124a6 Mon Sep 17 00:00:00 2001
From: Alex Suykov <alex.suykov@gmail.com>
Date: Wed, 31 Jan 2018 03:01:46 +0200
Subject: [PATCH] udevmod: increase event buffer size

---
 src/udev/udevmod.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/udev/udevmod.h b/src/udev/udevmod.h
index 5267176..d216cf5 100644
--- a/src/udev/udevmod.h
+++ b/src/udev/udevmod.h
@@ -58,7 +58,7 @@ struct top {
 
 	uint sep;
 	uint ptr;
-	char uevent[512+2];
+	char uevent[1024+2];
 
 	char saveid[16];
 };
-- 
2.16.1

